packages/next/build/webpack/config/blocks/css/loaders/utils/base62.ts

```diff
@@ -0,0 +1,78 @@
+const dictionary = [
+  '0',
```

Comment:
Starting a classname with a number is invalid, you'll likely need to tweak the approach here if you want to include numeric values.

Reply:
This is a pure base62 function, IMHO it's only responsibility is to encode a number into a string using base62. Class name related logic and checks belong to `SequentialIDGenerator` which essentially is an algorithm that feeds numbers starting from 0 to n to the base62 function. Skipping the ones that could produce a string starting with a numeric character when fed to base62.

Reply:
I refactored some code and made a couple of useful descriptive notes on the algorithm for an easier review.